frontend/resourceadm/utils/resourceUtils/resourceUtils.ts

```diff
@@ -0,0 +1,76 @@
+import type { SupportedLanguageKey, ResourceTypeOption } from "app-shared/types/ResourceAdm";
```

Comment:
There is some hard-coded language text within this file that needs to be translated using i18n. :) Thats a separate PR.